docs/modules/globalfee.md

```diff
@@ -29,41 +31,40 @@ A global fees list must meet the following properties:
 
-Additionally, node operators may set additional minimum gas prices which can be larger than the _global_ minimum gas prices defined on chain.
+2. One of the entries in the global fees list has a zero amount, e.g., `0uatom`, and the corresponding denom, e.g., `uatom`, is not present in `minimum-gas-prices` in `app.toml`, or node operators may set additional `minimum-gas-prices` in `app.toml` also zero coins.
 
+### Global Fee Params: `BypassMinFeeMsgTypes` and `MaxTotalBypassMinFeeMsgGasUsage`
 
-### minimum-gas-prices
+Bypass minimum fee messages are messages that are exempt from paying fees. The above global fees and the below local `minimum-gas-prices` checks do not apply for transactions that satisfy the following conditions:
 
-The `minimum-gas-prices` config parameter allows node operators to impose additional requirements for minimum fees. The following rules apply:
-
-- The denoms in `min-gas-prices` that are not present in the global fees list are ignored. 
-- The amounts in `min-gas-prices` are considered only if they are greater than the amounts for the corresponding denoms in the global fees list. 
-
-## Bypass Fees Message Types
+- Contains only bypass message types defined in `BypassMinFeeMsgTypes`.
+- The total gas used is less than or equal to `MaxTotalBypassMinFeeMsgGasUsage`.
+- In case of non-zero transaction fees, the denom has to be a subset of denoms defined in the global fees list.
 
-Bypass messages are messages that are exempt from paying fees. The above global fees and `minimum-gas-prices` checks do not apply for transactions that satisfy the following conditions: 
+Starting from gaiad `v11.0.0`, `BypassMinFeeMsgTypes` and `MaxTotalBypassMinFeeMsgGasUsage` are part of global fee params and can be proposed at network level. The default `bypass-min-fee-msg-types=[
+"/ibc.core.channel.v1.MsgRecvPacket",
+"/ibc.core.channel.v1.MsgAcknowledgement",
+"/ibc.core.client.v1.MsgUpdateClient",
+"/ibc.core.channel.v1.MsgTimeout",
+"/ibc.core.channel.v1.MsgTimeoutOnClose"
+]` and default `maxTotalBypassMinFeeMsgGasUsage=1,000,000`
 
-- Contains only bypass message types, i.e., bypass transactions.
-- The total gas used is less than or equal to `MaxTotalBypassMinFeeMsgGasUsage`. Note: the current `MaxTotalBypassMinFeeMsgGasUsage` is set to `1,000,000`.
-- In case of non-zero transaction fees, the denom has to be a subset of denoms defined in the global fees list.
+From gaiad v11.0.0, nodes that have the `bypass-min-fee-msg-types` field in their `app.toml` configuration are **not utilized**. Therefore, node operators have the option to either leave the field in their configurations or remove it. Node inited by gaiad v11.0.0 or later does not have `bypass-min-fee-msg-types` field in the `app.toml`.
```

Comment:
**Reminder** 
We should add a warning to the log messages if this is found in the config and document in the release notes clearly, otherwise people will be confused.